cpp/src/Slice/Parser.cpp

```diff
@@ -1003,6 +998,31 @@ Slice::Contained::parseFormatMetaData(const list<string>& metaData)
     return result;
 }
 
+bool
+Slice::Contained::isDeprecated(bool checkParent) const
+{
+    const string deprecate = "deprecate";
```

Comment:
Even though your functions are named "isDeprecated", you didn't add "deprecated" as an alias for the "deprecate" metadata?

Reply:
Yes, but only because `isDeprecate` isn't a sensible thing to say IMO. Nothing to do with our eventual renaming.
I plan to add the alias in my next PR, alongside removing the dispatch side.